runtime/include/DataView.hpp

```diff
@@ -15,6 +15,7 @@
 #pragma once
 
 #include <Exception.hpp>
+#include <vector>
```

Comment:
Minor point, but our convention is to put system-level/STL includes at the top, followed by project-wide/external-package includes and then local headers last. Have a look at our guidance docs here: https://github.com/PennyLaneAI/guidance-docs/blob/master/development/cpp-dev-guidelines.md#include-file-use.

```suggestion
#include <vector>

#include <Exception.hpp>
```

Reply:
Good catch, sorry I missed that!